### PR TITLE
Add linting to CI

### LINKED ISSUE #29: Add eslint to CI
It would be awesome to add eslint to the travis build.

### PR DESCRIPTION
**Pull request by webPapaya**
This PR closes #29 and adds linting to the CI.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,4 +5,4 @@ node_js:
   - '4.3'
   - '4.2'
   - '4.0'
-script: make dist && make test-dist
+script: make dist && make test-dist && make lint
```
Comment (danieljuhl) [inline .travis.yml]: Could you please do the linting first... thanks

Comment (webPapaya) [inline .travis.yml]: sure

**Comment by webPapaya** [comment]
I think if PR #28 is merged the CI will be green on this PR as well

File: .travis.yml [hunk 2]
```diff
@@ -5,4 +5,4 @@ node_js:
   - '4.3'
   - '4.2'
   - '4.0'
-script: make dist && make test-dist && make lint
+script: make dist && make lint && make test-dist
```
Comment (danieljuhl) [inline .travis.yml]: I think it should be ` make lint && make dist && make test-dist`, as linting should be the very first task

**Comment by danieljuhl** [comment]
One last thing, are you able to squash these commits? Thanks...

**Comment by webPapaya** [comment]
Should be done.

**Pull request merged**